packages/phoenix-ng/projects/phoenix-app/src/app/sections/atlas/atlas.component.spec.ts

```diff
@@ -1,15 +1,35 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { AtlasComponent } from './atlas.component';
-import { AppModule } from '../../../app/app.module';
+import { EventDisplayService } from '../../../../../phoenix-ui-components/lib/services/event-display.service';
 
 describe('AtlasComponent', () => {
   let component: AtlasComponent;
   let fixture: ComponentFixture<AtlasComponent>;
 
+  const mockEventDisplay = {
+    init: jest.fn(),
+    loadGLTFGeometry: jest.fn(),
+    getLoadingManager: jest.fn().mockReturnThis(),
+    addProgressListener: jest.fn().mockImplementation(() => {
+      component.loadingProgress = 100;
+    }),
+    addLoadListenerWithCheck: jest.fn().mockImplementation(() => {
+      component.loaded = true;
+    }),
+    getURLOptionsManager: jest.fn().mockReturnThis(),
+    getURLOptions: jest.fn().mockReturnThis(),
+    get: jest.fn().mockReturnThis(),
+  };
+
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [AppModule],
+      providers: [
```

Comment:
🤔 I don't think any significant difference is there. 

[They say](https://jestjs.io/docs/migration-guide) - "If you are using Jasmine, or a Jasmine like API (for example [Mocha](https://mochajs.org/)), Jest should be mostly compatible, which makes it less complicated to migrate to."  

Jest is not rendering those components. Since we only needed a mocked EventDisplayService, I removed this import.